DiamondLightSource/web-ui-components
# [LIMS-1373] Render with canvas in APNGViewer

**JIRA ticket**: [LIMS-1373](https://jira.diamond.ac.uk/browse/LIMS-1373)

**Summary**:

Use `canvas` to draw APNG frames, rather than the native `img` object

**Changes**:
- Draw directly on canvas instead of using `img` DOM element
- Remove `autoplay` prop

**To test**:
- Run `yarn storybook`, navigate to `visualisation/APNGViewer`
- Ensure APNG image is rendered correctly and responds to changes in props
